spec/components/work_history_review_component_spec.rb

```diff
@@ -97,39 +99,143 @@
-        Timecop.freeze(Time.zone.local(2019, 10, 1, 12, 0, 0)) do
-          result = render_inline(described_class.new(application_form: application_form, editable: false))
+        result = render_inline(described_class.new(application_form: application_form, editable: false))
 
-          expect(result.css('.govuk-summary-list__actions').text).not_to include(t('application_form.work_history.break.enter_label'))
-        end
+        expect(result.css('.govuk-summary-list__actions').text).not_to include(t('application_form.work_history.break.change_label'))
+      end
+    end
+
+    context 'when the work breaks feature flag is on, there are existing individual breaks and editable' do
+      it 'renders component with change and delete links' do
+        FeatureFlag.activate('work_breaks')
+        break1 = build_stubbed(:application_work_history_break, start_date: february2019, end_date: april2019)
+        breaks = [break1]
+        application_form = build_stubbed(
+          :application_form,
+          work_history_breaks: nil,
+          application_work_history_breaks: breaks,
+        )
+
+        result = render_inline(described_class.new(application_form: application_form))
+
+        expect(result.text).to include('Delete entry for break between February 2019 and April 2019')
+        expect(result.text).to include('Change description for break between February 2019 and April 2019')
+        expect(result.text).to include('Change dates for break between February 2019 and April 2019')
+      end
+    end
+
+    context 'when the work breaks feature flag is on, there are existing individual breaks and not editable' do
+      it 'renders component without change and delete links' do
+        FeatureFlag.activate('work_breaks')
+        break1 = build_stubbed(:application_work_history_break, start_date: february2019, end_date: april2019)
+        breaks = [break1]
+        application_form = build_stubbed(
+          :application_form,
+          work_history_breaks: nil,
+          application_work_history_breaks: breaks,
+        )
+
+        result = render_inline(described_class.new(application_form: application_form, editable: false))
+
+        expect(result.text).not_to include('Delete entry for break between February 2019 and April 2019')
+        expect(result.text).not_to include('Change description for break between February 2019 and April 2019')
+        expect(result.text).not_to include('Change dates for break between February 2019 and April 2019')
+      end
+    end
+
+    context 'when the work breaks feature flag is on and work history breaks has a value' do
+      it 'renders component with capture all work breaks and without individual breaks' do
+        FeatureFlag.activate('work_breaks')
+        application_form.work_history_breaks = 'I was sleeping.'
+        application_form.application_work_history_breaks = []
+
+        result = render_inline(described_class.new(application_form: application_form))
```

Comment:
You could create a helper method for this inside of this spec; allow passing in a few arguments which will let you reuse it for all the specs. Would be DRY-er, WDYT?

```ruby
result = render(feature_active: true, explanation: 'I was sleeping')

# ...

def render(feature_active: false, explanation: nil, breaks: [])
  if feature_active
    FeatureFlag.activate('work_breaks')
  else
    FeatureFlag.deactivate('work_breaks')
  end
  application_form.work_history_breaks = explanation
  application_form.application_work_history_breaks = breaks
 
  result = render_inline(described_class.new(application_form: application_form))
end
```